pvanalytics/quality/irradiance.py

```diff
@@ -309,3 +310,35 @@ def check_irradiance_consistency_qcrad(ghi, solar_zenith, dhi, dni,
                              bounds=bounds['low_zenith']))
 
     return consistent_components, diffuse_ratio_limit
+
+
+def clearsky_limits(measured, clearsky, csi_max=1.1):
+    """Identify irradiance measurements greater than clearsky values.
```

Comment:
Here, we "identify...greater than"  The function returns True when a value is less than or equal to.  Do we want to return True if greater than, or keep this behavior?

Reply:
I think we want to keep the current behavior. It matches the semantics of the qcrad validation functions in this module which return True for values that are physically plausible.

To make the documentation more clear, perhaps it should say "Identify irradiance measurements that do not exceed clearsky values"